### DISCUSSION TITLE
Beginner mod user here, how do I download this?

### DISCUSSION THREAD

**Discussion by willum123** [Doenerstyle/Invasion-Mod#4] [Q&A]
I have played minecraft for a while but could never figure out how to put mods on my computer until recently, and I have only used curseforge for mods until now. I remembered this mod from watching youtube videos when I was younger and really really want to play it but I cannot find it on forge and cannot figure out how to put it on my computer otherwise. If someone could point me in the direction or a good guide for how to install this mod or similar mods that would be really helpful. Thank you!

**Comment by Doenerstyle** [comment]
Hello there!
First of all, thank you for your interest in this mod! :)
Installing Forge mods is, for the vast majority of mods, as simple as installing Forge itself and then just dropping the mods' .jar file into the "mods" folder in your Minecraft directory.

If I understood you correctly, you're using the CurseForge App to play modded Minecraft. In that case, you can install any modpack/combination of mods for Minecraft 1.7.10, then simply download the Invasion Mod in the Releases section here on GitHub, and put that downloaded .jar file into the "mods" folder of the according modpack on your hard drive. This works the same for very most mods!
A short side-note: pretty much all 1.7.10 mods are for Forge. CurseForge is nothing like a "different Forge", but really just the name of a platform. So you can look for Forge mods on multiple websites _including_ CurseForge, but they're all made for Forge (for 1.7.10 and earlier at least).

If you need any further help, feel free to ask more specifically. There's also plenty of guides/tutorials on installing Forge 1.7.10 mods online, but I'm happy to help if you prefer that. :) You can also find my discord on my profile if that's more appealing to you.

Once you've got the mod running, I'd be happy to hear about any suggestions you might have or bugs/problems you encountered. Other than that, I hope you'll have lots of fun with this mod! :D
